src/international/creepOrganizer.ts

```diff
@@ -90,6 +95,16 @@ InternationalManager.prototype.creepOrganizer = function () {
         commune.creepsFromRoomAmount += 1
     }
 
+    // Loop through all of my creeps
+
+    for (const creepName in Memory.creeps) {
+        try {
+            processSingleCreep(creepName);
+        } catch (err) {
+            console.log('Exception processing creep named: ' + creepName + '. ' + err + err)
```

Comment:
Please remove this double error log

Reply:
Also use our customLog function instead of console.log

Reply:
done.   I've also enabled the code formatter.